production.ini

```diff
@@ -25,28 +28,22 @@ port = 6543
 # Begin logging configuration
 
 [loggers]
-keys = root, documentstore, sentry
+keys = root, documentstore
```

Comment:
não seria `kernel` no lugar de `documentstore`?

Reply:
O nome do pacote continua sendo `documentstore`. Apenas o repositório do GitHub foi renomeado para `kernel`. Talvez seja uma boa idéia dar mais um passo e renomear o pacote também.

Reply:
acho que o quanto antes for feito esta mudança é melhor